src/cli/cli.cpp

```diff
@@ -7881,6 +7857,105 @@ template <> otError Interpreter::Process<Cmd("trel")>(Arg aArgs[])
 #endif
 
 #if OPENTHREAD_FTD || OPENTHREAD_CONFIG_TMF_NETWORK_DIAG_MTD_ENABLE
+
+template <> otError Interpreter::Process<Cmd("vendor")>(Arg aArgs[])
+{
+    Error error = OT_ERROR_INVALID_ARGS;
+
+    /**
+     * @cli vendor name
+     * @code
+     * vendor name
+     * Google Nest
```

Comment:
Can you make the output of the get command examples be the same as what is being set in the other command example?  That way it's more clear which set/get go together, since you have to separate them.

Ideally for all 3 groups of commands (name, model, swversion)

Reply:
Good point. Will update in next push.

Reply:
Updated in new push